programs/adapter-lido/src/lib.rs

```diff
@@ -0,0 +1,124 @@
+
+        Ok(())
+    }
+
+    pub fn withdraw<'a, 'b, 'c, 'info>(
+        ctx: Context<'a, 'b, 'c, 'info, Action<'info>>,
+        input: Vec<u8>,
+    ) -> Result<()> {
+        // Get Input
+        let mut input_bytes = &input[..];
+        let input_struct = WithdrawInputWrapper::deserialize(&mut input_bytes)?;
+
+        msg!("Input: {:?}", input_struct);
+        
+        let withdraw_accounts = if input_struct.instruction == 23 {
+            vec![
+                AccountMeta::new(ctx.remaining_accounts[0].key(), false),
+                AccountMeta::new_readonly(ctx.remaining_accounts[1].key(), true),
+                AccountMeta::new(ctx.remaining_accounts[2].key(), false),
+                AccountMeta::new(ctx.remaining_accounts[3].key(), false),
+                AccountMeta::new_readonly(ctx.remaining_accounts[4].key(), false),
+                AccountMeta::new(ctx.remaining_accounts[5].key(), false),
+                AccountMeta::new(ctx.remaining_accounts[6].key(), true),
+                AccountMeta::new_readonly(ctx.remaining_accounts[7].key(), false),  
+                AccountMeta::new(ctx.remaining_accounts[8].key(), false),
+                AccountMeta::new_readonly(ctx.remaining_accounts[9].key(), false),
+                AccountMeta::new_readonly(ctx.remaining_accounts[10].key(), false),
+                AccountMeta::new_readonly(ctx.remaining_accounts[11].key(), false),
+                AccountMeta::new_readonly(ctx.remaining_accounts[12].key(), false),
+            ]
+        } else {
+            vec![
+                AccountMeta::new(ctx.remaining_accounts[0].key(), false),
+                AccountMeta::new_readonly(ctx.remaining_accounts[1].key(), true),
+                AccountMeta::new(ctx.remaining_accounts[2].key(), false),
+                AccountMeta::new(ctx.remaining_accounts[3].key(), false),
+                AccountMeta::new_readonly(ctx.remaining_accounts[4].key(), false),
+                AccountMeta::new(ctx.remaining_accounts[5].key(), false),
+                AccountMeta::new(ctx.remaining_accounts[6].key(), true),
+                AccountMeta::new_readonly(ctx.remaining_accounts[7].key(), false),
+                AccountMeta::new_readonly(ctx.remaining_accounts[8].key(), false),
+                AccountMeta::new_readonly(ctx.remaining_accounts[9].key(), false),
+                AccountMeta::new_readonly(ctx.remaining_accounts[10].key(), false),
+                AccountMeta::new_readonly(ctx.remaining_accounts[11].key(), false),
+            ]
+        };
+
+        let withdraw_ix = Instruction { program_id: ctx.accounts.base_program_id.key(), accounts: withdraw_accounts, data: input };
+        
+        invoke(&withdraw_ix, ctx.remaining_accounts)?;    
+        
+        let deactivate_ix = stake::instruction::deactivate_stake(&ctx.remaining_accounts[1].key(), &ctx.remaining_accounts[6].key());
+        invoke(&deactivate_ix, &vec![ctx.remaining_accounts[1].clone()])?;    
+
+        Ok(())
```

Comment:
Output result is missing. Need to add `OutputWrapper` can refer to [Tulip]( https://github.com/DappioWonderland/adapter-programs/blob/master/programs/adapter-tulip/src/lib.rs#L224)

Reply:
Done :)